acceptance-tests/src/test/resources/features/UKDrivingLicence/DVADrivingLicence.feature

```diff
@@ -11,7 +11,7 @@ Feature: DVA Driving Licence Test
     And I set the document checking route
     And I see a form requesting DVA LicenceNumber
 
-  @DVADrivingLicence_test @build @staging @integration @smoke
+  @DVADrivingLicence_test @build @staging @integration @smoke @direct
```

Comment:
You can put this over the feature at the top so it applies to all

Reply:
Was thinking about that I don't think we should do the validation tests when using that tag which is why I have added it to the individual tests

Reply:
@direct tag should be just to test anything that does a direct connection